12-target-details.tex

```diff
@@ -301,6 +301,46 @@ \section{Neko}
 
 \section{PHP}
 \label{target-php}
+\state{NoContent}
+
+\subsection{Getting started with PHP}
+\label{target-php-getting-started}
+
+Haxe can be a powerful tool for developing PHP applications. Let's look at our first example showing the toolchain. 
+
+Create a new folder and save this class as \ic{Main.hx}.
+
+\haxe{assets/HelloPHP.hx}
+
+To compile, either run this from the command line:
+
+\begin{lstlisting}
+haxe -php bin -main Main
+\end{lstlisting}
+
+.. or create and run (double-click) a file called \ic{compile.hxml}. In this example the hxml-file should be in the same directory as the example class.
```

Comment:
Ellipsis in technical documents are not good, but having only two dots is even worse. ;)

Can we restructure this?